lib/license.js

```diff
@@ -100,57 +106,33 @@ function fixCli(pkgPath) {
       case '(Artistic-2.0 OR LicenseRef-LICENSE.md)': // OBSOLETE
```

Comment:
There are certainly other ways to do it, but the advantages offered by them aren't worth the code churn at this point. This version is already readable, and that's generally the highest priority.